risc0/cargo-risczero/src/commands/build_guest.rs

```diff
@@ -98,7 +98,7 @@ impl BuildGuest {
         let rustflags_env = &[("RUSTFLAGS", rustflags.as_str())];
 
         let build = DockerFile::new()
-            .from_alias("build", "risczero/risc0-guest-builder:v0.17")
+            .from_alias("build", "risczero/risc0-guest-builder:v0.18")
```

Comment:
I think this is related to the first screenshot you posted and should not be updated. @flaub or @mothran would know better than me though.
```suggestion
            .from_alias("build", "risczero/risc0-guest-builder:v0.17")
```